README.md

```diff
@@ -1 +1,161 @@
+1. For building your own WordPress in Docker: clone this repository and place the WordPress theme you want to use into the `var/www/html/content/themes` directory. Develop locally using the `local-compose.yml` file, then build your Docker image and run those in the cloud with your own `docker-compose.yml` file that specifies your custom image.
+
+The instructions below will get you set up to run containers on [Triton](https://www.joyent.com/), or anywhere that supports the Autopilot Pattern.
+
+### Getting started on Triton
+
+1. [Get a Joyent account](https://my.joyent.com/landing/signup/) and [add your SSH key](https://docs.joyent.com/public-cloud/getting-started).
+1. Install the [Docker Toolbox](http://docker.com/toolbox) (including `docker` and `docker-compose`) on your laptop or other environment, as well as the [Joyent Triton CLI](https://www.joyent.com/blog/introducing-the-triton-command-line-tool)
+1. [Configure Docker and Docker Compose for use with Joyent](https://docs.joyent.com/public-cloud/api-access/docker):
+
+```bash
+curl -O https://raw.githubusercontent.com/joyent/sdc-docker/master/tools/sdc-docker-setup.sh && chmod +x sdc-docker-setup.sh
+./sdc-docker-setup.sh -k us-east-1.api.joyent.com <ACCOUNT> ~/.ssh/<PRIVATE_KEY_FILE>
+```
+
+### Configure your environment
+
+Check that everything is configured correctly by running `./setup.sh`. You'll need an SSH key that has access to Manta, the object store where the MySQL backups are stored. Pass the path of that SSH key as `./setup.sh ~/path/to/MANTA_SSH_KEY`. The script will create an `_env` file that names the variables that you will need to run your WordPress environment.
+
+#### Manta Settings
+
+The script will set defaults for almost every config variable, but the Manta config is required and must be set manually. The two most important variables there are:
+
+```
+MANTA_BUCKET= # an existing Manta bucket
+MANTA_USER= # a user with access to that bucket
+```
+
+The MySQL container will take a backup during its `preStart` handler and periodically while running. Configure these Manta settings to specify how and where this backup is stored. Here you need to specify the `MANTA_USER`, and also the `MANTA_BUCKET` where the backups will be stored.
+
+#### WordPress configuration
+
+The setup script will set working defaults for the entire WordPress configuration. The defaults will work for a quick "hello world" experience, but you'll probably want to set your own values for many fields.
+
+```
+# Environment variables for for WordPress site
+WORDPRESS_URL=http://my-site.example.org/
+WORDPRESS_SITE_TITLE=My Blog
+WORDPRESS_ADMIN_EMAIL=user@example.net
+WORDPRESS_ADMIN_USER=admin
+WORDPRESS_ADMIN_PASSWORD=<random string>
+WORDPRESS_ACTIVE_THEME=twentysixteen
+WORDPRESS_CACHE_KEY_SALT=<random string>
+#WORDPRESS_TEST_DATA=true # uncomment to import a collection of test content on start
```

Comment:
We should should add a note about the source of the test data in the text below.